src/auth/src/credentials/user_credential.rs

```diff
@@ -37,8 +37,12 @@ where
     }
 
     async fn get_headers(&mut self) -> Result<Vec<(HeaderName, HeaderValue)>> {
-        // TODO(#442) - implementation in progress
-        Err(CredentialError::new(false, BoxError::from("unimplemented")))
+        let token = self.get_token().await?;
+        let mut value = HeaderValue::from_str(&format!("{} {}", token.token_type, token.token))
+            .map_err(|e| CredentialError::new(false, e.into()))?;
+        value.set_sensitive(true);
+        let headers = vec![(AUTHORIZATION, value)];
+        Ok(headers)
```

Comment:
I would probably just write:

```suggestion
        Ok(vec![(AUTHORIZATION, value)])
```

But both seem fine.

Reply:
Fixed.